### PR TITLE
feat(youtube): migrate YouTube shortcode to use `webdev-infra`

### PR DESCRIPTION
**Pull request by MichaelSolati**

### CODE REVIEW

File: src/styles/components/_youtube.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //
 // =============================================================================
 
+.youtube,
 .w-youtube {
```
Comment (samthor) [inline src/styles/components/_youtube.scss:9]: Should we drop `w-youtube` since the shared code doesn't generate that?

**Review by samthor — APPROVED** [review]
I think you keep clobbering `package-lock.json` because you have an old version of NPM, right? I don't really care but it keeps getting bigger/smaller.

**Comment by MichaelSolati** [comment]
> I think you keep clobbering `package-lock.json` because you have an old version of NPM, right? I don't really care but it keeps getting bigger/smaller.

Lol, you're right. I think it's cause i swap nvm also. Uhm, should i upgrade and update the package lock?

File: src/styles/components/_youtube.scss (hunk 1 above)
Comment (MichaelSolati) [reply src/styles/components/_youtube.scss:9]: It was used for the livestream player, but ill modify that.

**Pull request merged**